facilitation_guide/learning_objectives.rmd

```diff
@@ -1,21 +1,18 @@
+
 Session 1 - Our Vision  
-1.	Explain how Modeling to Learn seeks to align the health care team in working together to improve quality of care.  
-2.	Articulate a team need that MTL will address.  
-3.	Recognize the interdependence of quality of work life and quality of care.  
-4.	Describe how MTL helps providers see and understand their work within VA data, standards and initiatives.  
+1.	Explain how Modeling to Learn will help align the team together to improve both quality of care amd quality of work life
+2.	Identify a specific team need that MTL will address  
+3.  Make VA data and intiatives transparent in the context of providers' work
```

Comment:
Remember that LOs are not the facilitator's objectives to cover during the session, but should complete the sentence, "After the session, the participant will be able to...". So 1 and 2 can translate to that, but 3 probably needs tweaking.